usaspending_api/etl/elasticsearch_loader_helpers/controller.py

```diff
@@ -115,31 +122,34 @@ def run_deletes(self):
         client = instantiate_elasticsearch_client()
         if self.config["data_type"] == "award":
             deleted_awards(client, self.config)
-        else:
+        elif self.config["data_type"] == "transaction":
             deleted_transactions(client, self.config)
+        else:
+            raise RuntimeError(f"No delete function implemented for type {self.config['data_type']}")
 
 
-def extract_transform_load(worker):
+def extract_transform_load(task):
     if abort.is_set():
-        logger.info(format_log(f"{worker.name} became lost on the way over."))
+        logger.info(format_log(f"{task.name} partition was skipped due to previous error"))
```

Comment:
Should log at `logger.warning(...)`. Would also be helpful to include partition number as well as task name in here too.

Reply:
k